scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#268.sql

```diff
@@ -0,0 +1,70 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+-- 1. Clean up animal_release table by deleting animal_release records with no corresponding surgery
+DELETE FROM animal_release ar
+WHERE NOT EXISTS (SELECT release_id FROM surgery s WHERE s.release_id = ar.id);
+
+---- Test results
+-- SELECT * FROM animal_release ar
+-- WHERE NOT EXISTS (SELECT release_id FROM surgery s WHERE s.release_id = ar.id);
+
+-- 2. When capture_location is null, assign value of release_location
+UPDATE animal_release
+SET capture_location = release_location
+WHERE capture_location IS NULL;
+
+---- Test results
+-- SELECT p.name AS project_name,
+-- transmitter_id,
+-- capture_locality,
+-- release_locality,
+-- ar.status AS release_status,
+-- ar.capture_location,
+-- ar.release_location
+-- FROM animal_release ar
+-- JOIN project p ON p.id = ar.project_id
+-- JOIN surgery s ON s.release_id = ar.id 
+-- JOIN sensor se ON se.tag_id = s.tag_id
+-- WHERE capture_location IS NULL
+-- ORDER BY project_name;
+
+-- 3. When release_location is null, assign value of capture_location
+UPDATE animal_release
+SET release_location = capture_location
+WHERE release_location IS NULL;
+
+---- Test results
+-- SELECT p.name AS project_name,
+-- transmitter_id,
+-- capture_locality,
+-- release_locality,
+-- ar.status AS release_status,
+-- ar.capture_location,
+-- ar.release_location
+-- FROM animal_release ar
+-- JOIN project p ON p.id = ar.project_id
+-- JOIN surgery s ON s.release_id = ar.id 
+-- JOIN sensor se ON se.tag_id = s.tag_id
+-- WHERE release_location IS NULL
+-- ORDER BY project_name;
+
+-- 4. Add constraints so that these two fields are mandatory in the database
+ALTER TABLE animal_release ALTER COLUMN capture_location SET NOT NULL;
```

Comment:
No worries, I'll comment that section out then and, once this PR has been merged and deployed, I will add a new groovy file with the Liquibase changelog in https://github.com/aodn/aatams/tree/master/grails-app/migrations. Is that you had in mind?